### ISSUE TITLE
rule "MD036": undefined method `match' for nil:NilClass (NoMethodError)

### ISSUE THREAD

**Issue by alexxa** [markdownlint/markdownlint#211]
```
/home/igulina/.gem/ruby/gems/mdl-0.4.0/lib/mdl/rules.rb:588:in `block (3 levels) in load': undefined method `match' for nil:NilClass (NoMethodError)
	from /home/igulina/.gem/ruby/gems/mdl-0.4.0/lib/mdl/rules.rb:583:in `each'
	from /home/igulina/.gem/ruby/gems/mdl-0.4.0/lib/mdl/rules.rb:583:in `block (2 levels) in load'
	from /home/igulina/.gem/ruby/gems/mdl-0.4.0/lib/mdl.rb:86:in `block (2 levels) in run'
	from /home/igulina/.gem/ruby/gems/mdl-0.4.0/lib/mdl.rb:84:in `each'
	from /home/igulina/.gem/ruby/gems/mdl-0.4.0/lib/mdl.rb:84:in `block in run'
	from /home/igulina/.gem/ruby/gems/mdl-0.4.0/lib/mdl.rb:74:in `each'
	from /home/igulina/.gem/ruby/gems/mdl-0.4.0/lib/mdl.rb:74:in `run'
	from /home/igulina/.gem/ruby/gems/mdl-0.4.0/bin/mdl:10:in `<top (required)>'
	from /home/igulina/bin/mdl:23:in `load'
	from /home/igulina/bin/mdl:23:in `<main>'
```

Markdown file I'm parsing: https://github.com/kubernetes/website/blob/master/content/en/docs/setup/turnkey/azure.md

**Comment by copperwalls** [comment]
@alexxa Line 587 is most likely the offending line.

I’ve opened a pull request. Hopefully it solves the issue.

**Comment by copperwalls** [comment]
@alexxa Just realized the “fix” will trigger a false positive. Sorry about that.

**Comment by copperwalls** [comment]
@alexxa Just opened another PR. See if it works for you.

**Comment by alexxa** [comment]
@copperwalls Yes, that PR #222 fixes the issue. Thank you!
